gcip/addons/aws/helper.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import boto3
+
+session = boto3.Session()
+sts = session.client("sts")
+
+
+def aws_account_id() -> str:
+    """
+    Returns the AWS account id.
+
+    Account ID is resolved in following precedence:
+    1. OS environment variable `AWS_ACCOUNT_ID`
+    2. From response of `sts.get_caller_identity()` function call with an initialized `boto3.client`
+
+    Returns:
+        str: AWS account id.
+    """
+    if os.environ.get("AWS_ACCOUNT_ID"):
+        return os.environ["AWS_ACCOUNT_ID"]
+
+    sts_response = sts.get_caller_identity()
+    return sts_response["Account"]
```

Comment:
`sts_response` is never used but as return value

```suggestion
    return sts.get_caller_identity()["Account"]
```

Reply:
Accepted